src/rewritable_units/element.rs

```diff
@@ -133,6 +133,14 @@ impl<'r, 't> Element<'r, 't> {
         self.start_tag.self_closing()
     }
 
+    /// Whether the element can have inner content.  In other words, is it either a self-closing
```

Comment:
> In other words, is it either a self-closing tag like `<foo />` or is an HTML void element which are implicitly self-closing, e.g. `<img>`.

This is the situation when the method returns false.  Can you clarify by saying something like "This returns `true` unless it is a self-closing tag ..." ?

Reply:
Changed to:

```
Whether the element can have inner content.  Returns `true` unless the element is an [HTML void
element](https://html.spec.whatwg.org/multipage/syntax.html#void-elements) or has a
self-closing tag (eg, `<foo />`).
```